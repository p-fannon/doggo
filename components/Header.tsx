import { Heading } from "@chakra-ui/react";

export default function Header() {
    return (
        <header className="z-50 sticky top-0">
            <div className="flex flex-nowrap justify-start justify-self-start min-w-full pl-3 py-1 shadow-md border-b-2 border-cyan-500 dark:border-slate-200">
                <Heading className="text-xl font-bold font-heading">ðŸ¦´ Doggo</Heading>
            </div>
        </header>
    )
}