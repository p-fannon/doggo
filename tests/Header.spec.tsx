import Header from '../components/Header'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'

describe('Document', () => {
    it('renders successfully', () => {
        const { container } = render(<Header />)

        expect(container).toMatchSnapshot()
        const heading = screen.getByRole('banner')
        expect(heading).toBeInTheDocument()
        expect(heading).toHaveTextContent('Doggo')
    })
})