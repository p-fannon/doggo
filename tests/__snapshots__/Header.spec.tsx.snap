// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Document renders successfully 1`] = `
<div>
  <header
    class="flex flex-nowrap justify-start justify-self-start min-w-full pl-3 py-1 shadow-md border-b-2 border-cyan-500 dark:border-slate-200 z-50"
  >
    <h2
      class="chakra-heading text-xl font-bold font-heading css-0"
    >
      ðŸ¦´ Doggo
    </h2>
  </header>
</div>
`;
